lang/python/CMakeLists.txt

```diff
@@ -0,0 +1,130 @@
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#  All rights reserved.
+#
+# See the file LICENSE for redistribution information.
+#
+
+project(python_swig C)
+
+# UseSWIG: Policy option to use the standard target name over a generated name.
+if(POLICY CMP0078)
+    cmake_policy(SET CMP0078 NEW)
+endif()
+
+# UseSWIG: Policy option to honor SWIG_MODULE_NAME.
+if(POLICY CMP0086)
+    cmake_policy(SET CMP0086 NEW)
+endif()
+
+find_package(SWIG REQUIRED)
+include(${SWIG_USE_FILE})
+
+set(python_libs)
+set(python_version)
+if("${CMAKE_VERSION}" VERSION_LESS_EQUAL "3.12.0")
+    # This method of finding python libs has been deprecated since version 3.12.
+    # If we are running with a greater CMake version, opt to use the Python3 package.
```

Comment:
Since Python3 is preferred starting from CMake version 3.12, would it be better to use `VERSION_LESS` here?

Reply:
Good find! I forgot to update it when I refactored the statement during development. Fixed it to `VERSION_LESS`